src/Microsoft.CSharp/tests/ArrayHandling.cs

```diff
@@ -31,25 +32,31 @@ public void ArrayTypeNames()
             d = new int[3];
             ex = Assert.Throws<RuntimeBinderException>(() => { string s = d; });
             Assert.Contains("int[]", ex.Message);
+        }
 
-            d = new int[3, 2, 1];
-            ex = Assert.Throws<RuntimeBinderException>(() => { string s = d; });
+        [Fact]
+        [SkipOnTargetFramework(TargetFrameworkMonikers.UapAot, "https://github.com/dotnet/corert/issues/3331")]
+        public void MultiDimArrayTypeNames()
+        {
+            dynamic d = new int[3, 2, 1];
+            RuntimeBinderException ex = Assert.Throws<RuntimeBinderException>(() => { string s = d; });
             Assert.Contains("int[,,]", ex.Message);
 
             d = Array.CreateInstance(typeof(int), new[] { 3, 2, 1 }, new[] { -2, 2, -0 });
             ex = Assert.Throws<RuntimeBinderException>(() => { string s = d; });
             Assert.Contains("int[,,]", ex.Message);
-
         }
 
         [Fact]
+        [SkipOnTargetFramework(TargetFrameworkMonikers.UapAot, "https://github.com/dotnet/corert/issues/3331")]
```

Comment:
Was this one really failing for that reason? I don't see any non-zero bounds or rank 1 multidims here.